### PR TITLE
[Feature] Support PHP 8

### PR DESCRIPTION
**Pull request by chris-doehring**
Hey there,

now that Laravel 6/7/8 supports php 8, this package can support php 8.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,6 +23,10 @@ matrix:
       env:
         - LARAVEL_VERSION=^8.0
         - PHPUNIT_VERSION=^9.0
+    - php: "8.0snapshot"
```
Comment (chris-doehring) [inline .travis.yml:26]: That's the only php 8 image Travis supports right now. Under `dist: trusty` there would be no php 8 image at all.

**Comment by lindyhopchris** [comment]
Thanks for submitting!

**Comment by chris-doehring** [comment]
@lindyhopchris Could you maybe create a new release which includes this change? This would make it possible to also allow php 8 for `cloudcreativity/laravel-json-api` :)

**Comment by lindyhopchris** [comment]
Thanks for the prompt! Just pushed the tag `3.2.0`

**Pull request merged**